addon/components/o-s-s/smart/immersive/logo.ts

```diff
@@ -0,0 +1,44 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { runSmartGradientAnimation } from '@upfluence/oss-components/utils/run-smart-gradient-animation';
+
+interface OSSSmartImmersiveLogoArgs {
+  icon?: string;
+  url?: string;
+  editable: boolean;
+  loading?: boolean;
+  oversize?: boolean;
+  onClick(): void;
+}
+
+export default class OSSSmartImmersiveLogoComponent extends Component<OSSSmartImmersiveLogoArgs> {
```

Comment:
as mentioned in my previous comment, lets have assertions on the fact that the icon parameter is using the proper format (if it's the one used ofc)